client/src/helpers/table/column.ts

```diff
@@ -0,0 +1,27 @@
+type OrderedColumn = {
+  label: string;
+}
+type OrderedColumns = OrderedColumn[];
+
+type OrderedColumnsArguments = {
+  orderedColumns: OrderedColumns;
+  schema: OrderedColumn[];
+};
+
+export const getOrderedColumns = ({ orderedColumns, schema }: OrderedColumnsArguments) => {
+  const output: OrderedColumns = [];
+
```

Comment:
the two loops are very similar. I would try to extract the logic into a function. IF it doesn't impact readability.